backend/src/main/java/com/celuveat/restaurant/application/RestaurantQueryService.java

```diff
@@ -101,4 +109,46 @@ private RestaurantQueryResponse toResponse(
     ) {
         return RestaurantQueryResponse.from(restaurantWithDistance, celebs, images);
     }
+
+    public List<RestaurantLikeQueryResponse> findAllRestaurantLikeByMemberId(Long memberId) {
+        OauthMember member = oauthMemberRepository.getById(memberId);
+        List<RestaurantLike> restaurantLikes = restaurantLikeRepository.findAllByMember(member);
+        List<Restaurant> restaurants = extractRestaurants(restaurantLikes);
+        List<Long> restaurantIds = extractRestaurantIds(restaurants);
+        List<Video> videos = videoRepository.findAllByRestaurantIdIn(restaurantIds);
+        Map<Long, List<Celeb>> celebs = mapVideoToCeleb(groupingVideoByRestaurant(videos));
+        List<RestaurantImage> images = restaurantImageRepository.findAllByRestaurantIdIn(restaurantIds);
```

Comment:
```suggestion
List<Video> videos = videoRepository.findAllByRestaurantIn(restaurants);
Map<Long, List<Celeb>> celebs = mapVideoToCeleb(groupingVideoByRestaurant(videos));
List<RestaurantImage> images = restaurantImageRepository.findAllByRestaurantIn(restaurants);
```

`restaurantIds`가 사용되는`Repository`들의 메서드를 `IdIn`가 아닌 `RestaurantIn`로 변경하면 Id를 추출하는 작업은 필요 없을거 같아

Reply:
반영했어! 👍